austrakka/components/sequence/__init__.py

```diff
@@ -90,6 +90,16 @@ def seq_list(
     help='The id of the sample which needs to have sequences purged.',
     multiple=False
 )
-def sequence_purge(sample_id: [str]):
+@option('--skip', cls=MutuallyExclusiveOption,
```

Comment:
Can we move these to the shared options? `force` in particular I can see being reused in the future. I think we can pass `cls` as a parameter